### ISSUE TITLE
Visually Assess Architecture Convergence 

### ISSUE THREAD

**Issue by ThomJ130** [sharifware/Large-Scale-Design-and-Analysis-of-Neural-Networks#2]
The system should generate and display the loss graph over iterations for $k$ number of neural networks with a given training set and $n$ epochs.

This tool will provide a means of comparing a few examples of the neural network to make sure it's converging and therefore working correctly. This is also for understanding how many iterations the neural network needs before being a valid solution, which can later be used to define the success criteria for creating the neural networks that will be used to generate the histograms/probability distributions.
